admin-dev/themes/new-theme/js/app/widgets/ps-tree/ps-tree-item.vue

```diff
@@ -188,17 +188,20 @@
       },
     },
     mounted() {
-      EventBus.$on('toggleCheckbox', (tag: any) => {
+      EventEmitter.on('toggleCheckbox', (tag: any) => {
```

Comment:
As we talked about it, the EventBus is not supported anymore, we could replace it with `useEventBus` from `vueuse` and scope it with keys, to avoid any side effects